back/src/domains/convention/use-cases/notifications/NotifyAgencyThatAssessmentIsCreated.ts

```diff
@@ -74,32 +110,50 @@ export class NotifyAgencyThatAssessmentIsCreated extends TransactionalUseCase<Wi
         status: assessment.status,
       });
 
-      await this.#saveNotificationAndRelatedEvent(uow, {
-        kind: "email",
-        templatedContent: {
-          kind: "ASSESSMENT_CREATED_WITH_STATUS_COMPLETED_AGENCY_NOTIFICATION",
-          recipients,
-          cc: [convention.signatories.beneficiary.email],
-          params: {
-            beneficiaryFirstName: convention.signatories.beneficiary.firstName,
-            beneficiaryLastName: convention.signatories.beneficiary.lastName,
-            businessName: convention.businessName,
+      for (const { email, role } of recipientsRoleAndEmail) {
```

Comment:
plutôt qu'une boucle for, tu as un utilitaire (qui fait la même chose, mais plus explicitement) : 
`executeInSequence`